src/diffusers/loaders/single_file_utils.py

```diff
@@ -518,7 +519,9 @@ def create_vae_diffusers_config(original_config, image_size, scaling_factor=None
     Creates a config for the diffusers based on the config of the LDM model.
     """
     vae_params = original_config["model"]["params"]["first_stage_config"]["params"]["ddconfig"]
-    scaling_factor = scaling_factor or original_config["model"]["params"]["scale_factor"]
+    scaling_factor = (
+        scaling_factor or original_config["model"]["params"]["scale_factor"] or LDM_VAE_DEFAULT_SCALING_FACTOR
+    )
```

Comment:
```suggestion
    if scaling_factor is None and "scale_factor" in original_config["model"]["params"]:
        scale_factor = original_config["model"]["params"]["scale_factor"]
    elif scaling_factor is None:
        scale_factor = LDM_VAE_DEFAULT_SCALING_FACTOR
```

Reply:
We can't be 100% sure that "scale_factor" is present in the dict

Reply:
This is safer, I agree.